CMakeLists.txt

```diff
@@ -63,6 +72,14 @@ if ( STRIP )
   string( APPEND CMAKE_C_FLAGS "-s" )
 endif(  )
 
+
+# include paths:
+#  - include/
+#  - the root directory so that we can e.g. #include "{external,backends}/..."
+include_directories(include .)
```

Comment:
Won't this break out-of-tree builds? I mean `mkdir build; cd build; cmake ..` which is now default in Fedora. I think it should use something like `$SRCDIR` instead of the "dot" dir.

Reply:
I just tried this out with cmake and building with `rpmbuild` everything seems to be working. I do think `${SRCDIR}` should be used for clarity.